v4_MIGRATION_GUIDE.md

```diff
@@ -733,12 +733,15 @@ public class ThreeDSecureActivity extends AppCompatActivity {
     ThreeDSecureAdditionalInformation additionalInformation = new ThreeDSecureAdditionalInformation()
         .accountId("account-id");
 
+    // ThreeDSecureRequest default version requested is now VERSION_2
+    // uiCustomization() for 3DS v2 has been renamed to v2UiCustomization()
+    // shippingMethod property is now an enum 
```

Comment:
I think what you have here works, but we could also put these notes above the whole 3DS code snippet. Maybe right before "To use the feature, instantiate a `ThreeDSecureClient`".

Feel free to copy from the [iOS migration guide](https://github.com/braintree/braintree_ios/blob/master/V5_MIGRATION.md#3d-secure), too. 🙂

Reply:
The iOS migration guide seemed clearer, so I updated to follow that 👍